src/main/scala/thurloe/Main.scala

```diff
@@ -8,15 +8,15 @@ import spray.can.Http
 
 import scala.concurrent.duration._
 
-object Boot extends App {
-
-  // we need an ActorSystem to host our application in
+object Main extends App {
+  // We need an ActorSystem to host our application in
   implicit val system = ActorSystem("on-spray-can")
 
   // create and start our service actor
   val service = system.actorOf(Props[ThurloeServiceActor], "demo-service")
 
   implicit val timeout = Timeout(5.seconds)
-  // start a new HTTP server on port 8080 with our service actor as the handler
-  IO(Http) ? Http.Bind(service, interface = "localhost", port = 8080)
+
+  // start a new HTTP server on port 8000 with our service actor as the handler
+  IO(Http) ? Http.Bind(service, interface = "0.0.0.0", port = 8000)
```

Comment:
I believe this isn't robust to failures, but :+1: for now.

https://groups.google.com/a/broadinstitute.org/forum/#!topic/dsde-engineering/xV96poHj5IA
broadinstitute/agora#1
broadinstitute/vault-common#7